README.md

```diff
@@ -3,12 +3,29 @@
 # Apache Kafka bridge
 
 This project provides a software component which acts as a bridge between different standard protocols and an [Apache Kafka](http://kafka.apache.org/) cluster.
-The current supported protocols are :
+The current supported protocols are:
 
 * [AMQP 1.0 (Advanced Message Queuing Protocol)](https://www.amqp.org/)
 * [HTTP 1.1 (Hypertext Transfer Protocol)](https://tools.ietf.org/html/rfc2616)
 
-It provides a different way to interact with Apache Kafka because the latter supports natively only custom (proprietary) protocol. Thanks to the bridge, all clients which can speak different standard protocols can connect to Apache Kafka cluster in order to send and receive messages to/from topics.
+It provides a different way to interact with Apache Kafka because the latter supports natively only custom (proprietary) protocol. 
+Thanks to the bridge, all clients which can speak different standard protocols can connect to Apache Kafka cluster in order to send and receive messages to / from topics.
+
+## Running the bridge
+
+### On Kubernetes and OpenShift
+
+You can use the [Strimzi Kafka operator](https://strimzi.io/) to deploy the HTTP Bridge on Kubernetes and OpenShift.
```

Comment:
"to deploy Kafka bridge" ... do we want to mention that using that way just the HTTP protocol support is included or not?

Reply:
I would leave it like this.

Reply:
but it's not the right name, it should be at least be "to deploy the Kafka bridge with the HTTP support"

Reply:
Better?